### PR TITLE
Remove FaceValue magic

### PR DESCRIPTION
**Pull request by mrmanc**
This removes dependence on switch ordering and enum value ordering.

### CODE REVIEW

File: exercises/java/src/main/java/cards/FaceValue.java [hunk 1]
```diff
@@ -1,39 +1,14 @@
 package cards;
 
-public enum FaceValue {
```
Comment (alectunbridge) [inline exercises/java/src/main/java/cards/FaceValue.java]: Looks good but I reckon if we're going this far then we might as well ditch the whole FaceValue class and just have it as a String attribute on Card?

Comment (mrmanc) [reply exercises/java/src/main/java/cards/FaceValue.java]: 🤷‍♂️  I’d be up for that. Was there an intention of introducing them to enums?

Comment (alectunbridge) [reply exercises/java/src/main/java/cards/FaceValue.java]: Nope let's just bin the whole class

Comment (mrmanc) [reply exercises/java/src/main/java/cards/FaceValue.java]: Done in a4f7c29e8587e9b04bd5f009bffd53993f52c00a

**Pull request merged**